core/templates/pages/skill-editor-page/editor-tab/skill-description-editor/skill-description-editor.directive.html

```diff
@@ -5,7 +5,7 @@
       This is the name of the skill as visible to learners and other creators.
     </div>
     <p ng-class="{'has-error': !tmpSkillDescription.length > 0}">
-      <input type="text" class="form-control protractor-test-skill-description-field" ng-if="canEditSkillDescription()"
+      <input type="text" aria-label="Skill Description Field" class="form-control protractor-test-skill-description-field" ng-if="canEditSkillDescription()"
```

Comment:
```suggestion
      <input type="text" aria-label="Skill Description" class="form-control protractor-test-skill-description-field" ng-if="canEditSkillDescription()"
```

So that it matches what we have done with Topics (core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html).